2019-11-26 16:02:29,474 - root - INFO - db_calls.py:200 - Obtaining registry data...
2019-11-26 16:02:29,474 - root - INFO - db_calls.py:64 - Connecting to database...
2019-11-26 16:02:30,119 - root - INFO - db_calls.py:275 - Importing data...
2019-11-26 16:02:37,987 - root - INFO - db_calls.py:220 - Obtaining source data...
2019-11-26 16:02:37,987 - root - INFO - db_calls.py:64 - Connecting to database...
2019-11-26 16:02:38,512 - root - INFO - db_calls.py:275 - Importing data...
2019-11-26 16:02:44,551 - root - INFO - data_processing.py:171 - Re-organising source data...
2019-11-26 16:02:45,389 - root - INFO - data_processing.py:171 - ...done
2019-11-26 16:02:45,443 - root - INFO - data_processing.py:279 - Re-organising registry data...
2019-11-26 16:02:53,452 - root - INFO - data_processing.py:321 - ...done
2019-11-26 16:02:54,495 - root - INFO - data_matching.py:171 - Starting matching...
2019-11-26 16:03:00,415 - root - ERROR - runfile.py:45 - Uncaught exception: 
NoneType: None
